README.md

```diff
@@ -1 +1,92 @@
 # i18nify-react
+
+_A React-Friendly Internationalization (i18n) State Manager_
```

Comment:
Let's not keep State Manager in the title right now, It could be anything in future.

Reply:
Removed.